build/tutorials-contest/liquid-avax/Backend/backend.js

```diff
@@ -0,0 +1,272 @@
+                    
+                    let importPTx = await pChain.issueTx(signedImportPTx);
+                    
+                    console.log("importPTx " + importPTx);
+                    
+                    fee = pChain.getDefaultTxFee();
+            
+                    amount = amount.sub(fee);
+                } else {
+                    // Error
+                }
+            })
+        } else {
+            // Throw an error
+        }
+    })
+}
+
+let binTools = BinTools.getInstance();
+
+async function importKeys() {
+    let buffer = Buffer.from(privateKey, 'hex')
+    let CB58Encoded = `PrivateKey-${binTools.cb58Encode(buffer)}`
+    xKeyChain.importKey(CB58Encoded);
+    cKeyChain.importKey(buffer);
+    pKeyChain.importKey(CB58Encoded);
+}
+
+let transactionStatus;
+
+async function waitForStatusX(transactionId) {
+    transactionStatus = await xChain.getTxStatus(transactionId);
+    return transactionStatus;
+}
+
+async function waitForStatusP(transactionId) { //Either X or P chain.
+    transactionStatus = (await pChain.getTxStatus(transactionId)).status;
+    return transactionStatus;
+}
+
+async function waitForStatusC(transactionId) {
+    transactionStatus = await cChain.getAtomicTxStatus(transactionId);
+    return transactionStatus;
+}
+
+async function pollTransaction(func, transactionID, resolve, reject) {
+    transactionStatus = await func(transactionID);
+    if (transactionStatus === "Rejected") {
+        reject(transactionStatus);
+    }
+    if (transactionStatus !== "Accepted") {
+        setTimeout(pollTransaction, 100, func, transactionID, resolve, reject);
+    } else {
+        resolve(transactionStatus);
+    }
+}
+
+let xAvaxAssetId, cAvaxAssetId;
+
+let avalancheInstance;
+
+async function getInformations() {
+    const ip = "localhost";
+    const port = 9650;
+    const protocol = "http";
+    const networkID = 5;
+    avalancheInstance = new Avalanche(ip, port, protocol, networkID);
+    
+    const path = '/ext/bc/C/rpc';
+    
+    xChain = avalancheInstance.XChain();
+    xKeyChain = xChain.keyChain();
+    xChainBlockchainID = xChain.getBlockchainID();
+    
+    cChain = avalancheInstance.CChain();
+    cKeyChain = cChain.keyChain();
+    cChainBlockchainID = cChain.getBlockchainID();
+    cChainAddress = cKeyChain.getAddressStrings();
+    
+    pChain = avalancheInstance.PChain();
+    pKeyChain = pChain.keyChain();
+    pChainBlockchainID = pChain.getBlockchainID();
+    pChainAddress = pKeyChain.getAddressStrings();
+    
+    xAvaxAssetId = binTools.cb58Encode((await xChain.getAssetDescription("AVAX")).assetID);
+    cAvaxAssetId = binTools.cb58Encode((await cChain.getAssetDescription("AVAX")).assetID);
+    
+    const web3Protocol = "wss";
+    const web3Path = '/ext/bc/C/ws';
+    
+    //web3 = new Web3(`${web3Protocol}://${ip}:${port}${web3Path}`);
+    web3 = new Web3("wss://api.avax-test.network/ext/bc/C/ws");
```

Comment:
Remove hardcoded path and instead use the interpolated string in the line above.